docs/source/basics/network-layer.md

```diff
@@ -207,14 +207,14 @@ class AppModule {
     httpLink: HttpLink,
     auth: Auth
   ) {
-    const http = httpLink.create({ uri: '/graphql' });
+    const http = this.httpLink.create({ uri: '/graphql' });
```

Comment:
Here should be just `httpLink`, no `this.httpLink` because it's an argument of the constructor.